app/lib/metric_util_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class HeatmapHelperTest < ActiveSupport::TestCase
```

Comment:
I think you forgot to change the name HeatmapHelperTest, but also we should use Rspec instead and have it in the /spec folder

Reply:
or at least the /test directory with Minitest

Reply:
👍